docs/api/api/api.txt

```diff
@@ -1948,6 +1948,11 @@ DELETE /staging/<staging_workflow_project>/workflow
   Delete a staging workflow associated to a given project.
 XmlResult: status_ok
 
+POST /staging/<staging_workflow_project>/workflow/accept_staging_projects
```

Comment:
I guess it's a little late now, but why you want to accept multiple staging projects through the API? Especially if this runs in background, there is really no reason to force XML onto the API user, I can quicker and easier script a loop using a single project.

Because the posted XML does not conform to the XML schema (which would require a group attribute, as such I doubt your schema is right - where is it anyway?), so e.g.

`/staging/<staging_workflow_project>/staging_projects/<staging_project>/accept` would be the easier route.

Reply:
For the validation check 0a586593e914b391c0a7f2c9cae6091ee0517fc2

Reply:
True. We could adjust this. It's not ideal, but given this is still beta it shouldn't be a problem.

Reply:
No, for this release it's okay.